server/core/config/raw/project.go

```diff
@@ -142,6 +145,16 @@ func validProjectName(name string) bool {
 	return nameWithoutSlashes == url.QueryEscape(nameWithoutSlashes)
 }
 
+func validPlanReq(value interface{}) error {
```

Comment:
Please test this in project_test.go

Reply:
I think https://github.com/runatlantis/atlantis/pull/2979/files#diff-c2d2795c6c7b0f0eb9693b7613f8a84d8945e1e2864f31acd5313dca60fa9a31R127 is exercising this function. Let me know if I'm mistaken.

Reply:
I don't see a unit test for the function `validPlanReq` tho

Reply:
Added test cases. I think this is resolved.